http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2087,6 +2439,38 @@ components:
           description: The number of minutes of idle time to elapse before the cluster is
             autopaused. If autopause is set to false, this value is disregarded.
             A value of 0 is equivalent to autopause being turned off.
+    CreateDiskRequest:
+      description: "Creates a new persistent disk"
+      type: object
+      properties:
+        labels:
+          type: object
+          description: The labels to be placed on the persistent disk. Of type Map[String,String]
+        size:
+          type: integer
+          description: The size of the persistent disk to be created in GB.
+        diskType:
+          type: string
+          description: The type of the persistent disk to be created. Can be "Standard" or "SSD".
+        blockSize:
+          type: integer
+          description: The block size of the persistent disk to be created in bytes.
+    UpdateDiskRequest:
+      description: "Updates an existing persistent disk"
+      type: object
+      properties:
+        labels:
+          type: object
+          description: The labels to be added or updated on the persistent disk. Of type Map[String,String]
+        newSize:
```

Comment:
I think terra ui preferred these names remain the same...like `size`, `diskType`, `blockSize`...

Reply:
+1